src/cli/litra-toggle.ts

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+
+import { findDevice, turnOff, turnOn } from "./../driver";
+
+try {
+  const device = findDevice();
+
+  if (!device) {
+    throw "Device not found";
+  }
+
+  const deviceArgKey = "-device";
```

Comment:
By convention, we use `ALL_CAPS` for constants like this - so could you change this one to `const DEVICE_ARG_KEY`? The same will apply to the next few lines.